lib/getPaths.js

```diff
@@ -11,7 +11,7 @@
  */
 module.exports = function getPaths(path) {
 	if (path === "/") return { paths: ["/"], segments: [""] };
-	const parts = path.split(/(.*?[\\/]+)/);
+	const parts = path.split(/(.*?[/]+)/);
```

Comment:
And this

Reply:
Same idea as this: https://github.com/webpack/enhanced-resolve/pull/436#discussion_r1826329289